src/ngraph/builder/norm.cpp

```diff
@@ -97,17 +98,24 @@ namespace ngraph
             return values + bias_node;
         }
 
-        shared_ptr<Node>
-            l2_norm(const Output<Node>& value, const AxisSet& reduction_axes, float bias)
+        shared_ptr<Node> l2_norm(const Output<Node>& value,
+                                 const AxisSet& reduction_axes,
+                                 float bias,
+                                 BiasMode bias_mode)
         {
             shared_ptr<Node> values{make_shared<op::Sum>(value * value, reduction_axes)};
 
             shared_ptr<Node> bias_node{
                 op::Constant::create(values->get_element_type(),
                                      values->get_shape(),
                                      vector<float>(shape_size(values->get_shape()), bias))};
-
-            return {make_shared<op::Sqrt>(values + bias_node)};
+            switch (bias_mode)
+            {
+            case BiasMode::MAX:
+                return {make_shared<op::Sqrt>(make_shared<op::Maximum>(values, bias_node))};
```

Comment:
I think we should use `{}` braces around this line for style consistency.

Reply:
Done